src/components/POLine/Item/MaterialTypesForm.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Field, FieldArray } from 'redux-form';
+import { Row, Col, Button, TextField } from '@folio/stripes/components';
+
+class MaterialTypesForm extends Component {
+  constructor(props) {
+    super(props);
+    this.renderForm = this.renderForm.bind(this);
```

Comment:
What is a purpose of binding arrow functions?
Babel transpiles it to
`constructor(props) {
   super(props); 
   this.renderForm = ({ fields }) => {...} `
Please, remove the binding or use 
`renderForm ({ fields }) {...}` and bind it in the constructor. The latter may be a little faster.